server.js

```diff
@@ -50,8 +50,8 @@ if (!process.env.GITHUB_USER) {
 
 var github = new GitHubApi({
   version: '3.0.0',
-  host: config.gheHost,
-  pathPrefix: config.ghePathPrefix,
+  host: process.env.GITHUB_HOST || config.gheHost,
+  pathPrefix: process.env.GITHUB_API_PREFIX || config.ghePathPrefix,
```

Comment:
Can you respect the existing naming so that we don't have two different ones?

GHE_PATH_PREFIX
GHE_HOST